synthesizer/program/src/finalize/input/mod.rs

```diff
@@ -17,7 +17,7 @@ mod parse;
 
 use console::{
     network::prelude::*,
-    program::{PlaintextType, Register},
+    program::{FinalizeType, Register},
 };
 
 /// An input statement defines an input argument to finalize, and is of the form
```

Comment:
Documentation 1 line down should say `finalize_type` instead of `plaintext_type`